### PR TITLE
Add a default, no-op prepare block

### PR DESCRIPTION
**Pull request by sharplet**

### CODE REVIEW

**Comment by sharplet** [comment]
Also decided to go ahead and add a doc comment.

File: Sources/Perform/Perform.swift [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+  ///     and destination view controller type.
+  ///
+  /// - parameter prepare:
+  ///     A function that will be invoked with the matching destination view
+  ///     view controller when `prepareForSegue(_:sender:)` is received. The
+  ///     default value for this parameter does nothing.
+  ///
+  /// - note:
+  ///     If no matching view controller is found in the destination
+  ///     view controller hierarchy, this method raise a fatal error
```
Comment (jakecraige) [inline Sources/Perform/Perform.swift]: Missing a word between "method" and "raise", maybe "will", or "raises"?

**Comment by jakecraige** [comment]
👍

File: Sources/Perform/Perform.swift (hunk 1 above)
Comment (sharplet) [reply Sources/Perform/Perform.swift]: Good catch 👍

**Pull request merged**